taipy/gui/gui.py

```diff
@@ -2621,3 +2637,16 @@ def stop(self):
 
     def _get_autorization(self, client_id: t.Optional[str] = None, system: t.Optional[bool] = False):
         return contextlib.nullcontext()
+
+    def set_favicon(self, favicon_path: t.Union[str, Path]):
+        """
```

Comment:
Same line, please. Just like the others
-> 
`"""Change the favicon for all clients.`